Cargo.toml

```diff
@@ -2,4 +2,5 @@
 members = ["bitcoin", "hashes", "internals"]
 
 [patch.crates-io.bitcoin_hashes]
-path = "hashes"
+git = "https://github.com/tcharding/rust-bitcoin"
+branch = "01-20-rename-hashes-methods"
```

Comment:
Why is this necessary?

Reply:
Messy isn't it. This is required because otherwise the secp dependency uses a different version of `hashes`. Even though the version _is_ this branch we are on the `path` dependency gives a different version.

Reply:
This is the problem I mention [over here](https://github.com/rust-bitcoin/rust-bitcoin/issues/1553#issuecomment-1410017923). Its caused by the fact that `path` dependency does not patch over git/branch dependency (it only patches over crates.io) so there is, as far as I can telll, no way to get the same version of `hashes` if we use git/branch deps (or tag as you suggest). Am I missing some other way? This really borks things because we can now no longer use `path` dependency, even once this merges, we have to release secp in order to start using `path` again. We can switch to using `master` (or a tagged commit on master) but then we have to update that continually.

Reply:
Why are you changing the `bitcoin_hashes` dep in the rust-secp branch at all? Just leave it pointing to crates, then the patch should do the right thing.

Reply:
Bother, I'm on a different machine and I forgot to pull before I force pushed. Have to re-implement the review suggestions from before - won't be long :)